test/unit/auth/auth.spec.ts

```diff
@@ -47,6 +49,15 @@ chai.use(chaiAsPromised);
 const expect = chai.expect;
 
 
+interface AuthTest {
```

Comment:
This file is bit complicated to review. Can I assume it's just running the existing tests in 2 test configs, with the tests specific to an auth implementation enclosed in if-blocks?

Reply:
Yeah, I completely understand. This is the same GitHub diff issue. This should no longer be an issue in the next PRs.
You are mostly right. Same tests are run for both. There are cases where I add specific tests for tenant specific behavior. For example:

- createCustomToken should throw unsupported tenant operation error
- verifyIdToken with token missing or mismatching tenant ID
- verifySessionCookie with missing or mismatching tenant ID
- importUsers with explicit mismatching tenant IDs
- confirm createSessionCookie calls verifyIdToken before making call to create a session cookie

I do this by checking `testConfig.Auth === TenantAwareAuth`